### DISCUSSION TITLE
How to change the height of the ckeditor

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tw5-tools#8] [Q&A]
Sorry for asking in the tiddlyclip repo. I want to know how to change the height of the ckeditor when I am editing the clipped html

**Comment by buggyj** [comment]
There is a class tiddler and the config tiddler - I don't have not done this before so you may need to experiment with the settings. The height of the editor is in the config.json and the height of the enclosing tiddler is controlled by the stylesheet tiddler.


[$ _plugins_bj_visualeditor_config.json.json.zip](https://github.com/buggyj/tw5-tools/files/8172802/_plugins_bj_visualeditor_config.json.json.zip)
